app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/settings/SettingsPanel.tsx

```diff
@@ -53,17 +57,22 @@ export const SettingsPanel = () => {
     : "Protected A";
 
   const handleUpdateClassification = useCallback(
-    (value: ClassificationType) => {
+    async (value: ClassificationType) => {
       if (value === "Protected B") {
         setDeliveryOption(DeliveryOption.vault);
       }
 
-      updateSecurityAttribute(value);
-
-      updateTemplateSecurityAttribute({
+      // Call sever action
+      const result = await updateTemplateSecurityAttribute({
         id,
         securityAttribute: value,
       });
+
+      if (!result.error) {
+        updateSecurityAttribute(value);
+      }
+
```

Comment:
Probably want to add a return in the above condition or similar. Currently the toast is always reached